e2e/runner/setup_sui.go

```diff
@@ -560,6 +563,90 @@ func (r *E2ERunner) suiTransferObjectToTSS(signer *zetasui.SignerSecp256k1, obje
 	r.suiExecuteTx(signer, tx)
 }
 
+// UpdateTSSAddressSui updates the TSS for Sui by issuing a new WithdrawCap and transferring it to the new TSS address.
+func (r *E2ERunner) UpdateTSSAddressSui(faucetURL string) {
```

Comment:
nit: if you could use the consistent function name convention to begin with `Sui` prefix, e.g. `SuiUpdateTSSAddress`

Reply:
The name is consistent with the other update functions 

	deployerRunner.UpdateTSSAddressForConnectorNative()
	deployerRunner.UpdateTSSAddressForERC20custody()
	deployerRunner.UpdateTSSAddressForGateway()
	deployerRunner.UpdateTSSAddressSolana
	deployerRunner.UpdateTSSAddressSui(conf.RPCs.SuiFaucet)

It would be better to just keep the name